src/plugins/orangehrmPimPlugin/Controller/File/EmployeePictureController.php

```diff
@@ -61,7 +58,7 @@ public function handle(Request $request)
             $empPicture = $this->getEmployeePictureService()->getEmpPictureByEmpNumber($empNumber);
```

Comment:
I want to minimize this call, as it goes to DB. want to reduce this call and cache the profile picture.

Reply:
https://github.com/orangehrm/orangehrm/pull/1674/files#diff-7a6b406d8bb4fdbbe35be9fa188826a121e49244e2da957cdeb91dc322f0328fR58-R67

DB call will not happen if the eTag is cached and the response is not modified